spec/forms/teacher_interface/alternative_name_form_spec.rb

```diff
@@ -8,9 +8,7 @@
   let(:application_form) { build(:application_form) }
 
   describe "#valid?" do
```

Comment:
```suggestion
  describe "validations" do
```

Since we're not strictly testing a method call directly.